src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+
+}
+
+body {
+  margin: 0;
+  padding-left: 8px;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  outline: 0;
+  width: 100%;
+  border-radius: 4px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  flex-shrink: 0;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+}
+
+.search-bar__input--big {
+  padding: 27px 0 26px 61px;
+  background: url("./images/Search.svg") 25px 24px no-repeat;
+  background-size: 19px;
+
+}
```

Comment:
there is no need to create this modifier, just transfer these styles to `.search-bar__input'